features/visitor_can_see_articles_on_index_page.feature

```diff
@@ -7,13 +7,14 @@ Feature: List articles on the landing page
     Background: 
         Given the following users exist
             | email          | password | role       |
-            | jocke@craft.se | password | journalist | 
+            | jocke@craft.se | password | journalist |
+            | per@mail.se    | password | journalist |
 
         And the following articles exist
-            | title                                     | lead                        | content                                                     | category     | user           |
-            | Voted best mead recipe                    | Restaurant wins prize       | Restaurant wins prize for best mead in Sweden               | Lifestyle    | jocke@craft.se |
-            | Ancient viking grave discovered           | Kids came across sword      | Kids come across sword protruding from the earth            | Breaking News| jocke@craft.se |
-            | Drinking wine improves general health     | Drink wine today!           | Studies show that wine is good for your heart               | Health       | jocke@craft.se |
+            | title                                 | lead                   | content                                          | category      | user           | approved |
+            | Voted best mead recipe                | Restaurant wins prize  | Restaurant wins prize for best mead in Sweden    | Lifestyle     | jocke@craft.se | true     |
```

Comment:
delete approved column, factory bot is handling it for value true

Reply:
yes thats right, done